### PR TITLE
feat: Use App SDK 3.1.

### PR DESCRIPTION
**Pull request by lenny-intel**
### CODE MAINTAINABILITY

- [x] **_Commit Message meets guidelines as indicated in the URL\*_**
    - <https://github.com/intel/AiCSD/blob/main/.github/CONTRIBUTING.md>
- [x] Tests for the changes have been added (required for bug fixes / features). **If not, state why not**
- [x] Added labels (these are git labels such as bug or enhancement)
- [ ] For any API changes or service changes, updates to the documentation were made (Swagger APIs & GitHub Docs). **If not, state why not**
- [ ] **_Every commit is a single defect fix and does not mix feature addition or changes\*_**

### What are the steps to test this PR contribution?

How does a reviewer know it works?

Are there any special instructions beyond standard workflow?

### QUALITY CHECKS

- [ ] **_Quality of code (At least one should be checked as applicable)\*_**
    - [ ] Commit Message meets guidelines
    - [ ] Code copyright is correct
    - [ ] PR changes adhere to industry practices and standards as well as domain or language specific anti-patterns
    - [ ] Adopted domain specific coding standards ([Go Coding Standards](https://go.dev/doc/effective_go))
    - [ ] Code is adequately commented and documented
    - [ ] Confusing logic is explained in comments
    - [ ] Error and exception code paths implemented correctly
    - [ ] Tracing output are minimized and logic
- [ ] **_Test coverage shows adequate coverage with required CI functional tests pass on all supported platforms\*_**
- [ ] **_Static code scan report shows zero critical issues\*_**

## Other information

### CODE REVIEW

File: ms-file-sender-oem/main.go [hunk 1]
```diff
@@ -63,7 +64,7 @@ func main() {
 		lc.Errorf("Retry on startup failed: %s", err.Error())
 	}
 
-	err = service.MakeItRun()
+	err = service.Run()
 	if err != nil {
 		lc.Errorf("MakeItRun returned error: %s", err.Error())
```
Comment (jim-wang-intel) [inline ms-file-sender-oem/main.go:69]:
```suggestion
		lc.Errorf("service Run returned error: %s", err.Error())
```

File: ms-file-watcher/main.go [hunk 2]
```diff
@@ -69,7 +70,7 @@ func main() {
 	wg.Add(1)
 	go fileWatcher.WatchFolders(ctx, wg, fileWatcher.Config)
 
-	err = service.MakeItRun()
+	err = service.Run()
 	if err != nil {
 		lc.Errorf("MakeItRun returned error: %s", err.Error())
```
Comment (jim-wang-intel) [inline ms-file-watcher/main.go:75]:
```suggestion
		lc.Errorf("service Run returned error: %s", err.Error())
```

**Review by jim-wang-intel — COMMENTED** [review]
LGTM, just some mirror thing on error messages

File: as-file-receiver-oem/main.go [hunk 3]
```diff
@@ -74,7 +75,7 @@ func main() {
 		os.Exit(-1)
 	}
 
-	if err := service.MakeItRun(); err != nil {
+	if err := service.Run(); err != nil {
 		lc.Errorf("MakeItRun returned error: %s", err.Error())
```
[the remark this answers is not in the document]
Comment (lenny-intel) [reply as-file-receiver-oem/main.go:79]: Good catch.

[the remark this answers is not in the document]
Comment (lenny-intel) [reply as-file-receiver-oem/main.go]: Fixed all occurences.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by ejlee3 — APPROVED** [review]
LGTM

**Pull request merged**